include/grpc/grpc.h

```diff
@@ -351,6 +351,23 @@ typedef struct grpc_op {
   } data;
 } grpc_op;
 
+/* Propagation bits: this can be bitwise or-ed to form propagation_mask for
+ * grpc_call */
+/** Propagate deadline */
+#define GRPC_PROPAGATE_DEADLINE ((gpr_uint32)1)
+/** Propagate census context */
+#define GRPC_PROPAGATE_CENSUS_CONTEXT ((gpr_uint32)2)
```

Comment:
This flag is somewhat redundant with the census state, as reflected in the return value from census_enabled() - this is what I was intending to use as a signal for whether we should do context propagation or not. We can certainly have both signals, but it could be confusing.